packages/core/src/multiline-input/MultilineInput.css

```diff
@@ -69,6 +69,7 @@
   --multilineInput-background-hover: var(--salt-status-error-background);
   --multilineInput-borderColor: var(--salt-status-error-borderColor);
   --multilineInput-outlineColor: var(--salt-status-error-borderColor);
+  --multilineInput-background-readonly: var(--salt-status-error-background);
```

Comment:
Assuming this is fixing code not implementing design correctly before. reminder to add changeset.

Reply:
@origami-z , can you help me on how to add it to changeset? I have also enabled border radius for multiline input.

Reply:
Added one (under `/.changeset`). You can add yourself as well. It will become the changelog / release note when released.

Reply:
Also given adding rounded corner support, let's also enable visual regression for it. See https://github.com/jpmorganchase/salt-ds/blob/c013b93ca7147aed7ee2055da6532599a743b238/packages/core/stories/text/text.qa.stories.tsx#L93-L108

Reply:
Added the above